bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/hw/HwSerialBridgeHandler.java

```diff
@@ -99,9 +100,16 @@ public void setDiscoveryService(HwDiscoveryService discoveryService) {
     }
 
     private void openConnection() {
+        SerialPortIdentifier portIdentifier;
+
+        portIdentifier = serialPortManager.getIdentifier(serialPortName);
```

Comment:
It's now possible to assign the variable where it is declared:

```suggestion
        SerialPortIdentifier portIdentifier = serialPortManager.getIdentifier(serialPortName);
```

Reply:
Ah, yes. Missed those.